Porter/Multi.elm

```diff
@@ -0,0 +1,143 @@
+module Porter.Multi
+    exposing
+        ( Request
+        , request
+        , fromSimple
+        , andThen
+        , andThenResult
+        , map
+        , map2
+        , map3
+        , send
+        )
+
+{-| With `Porter.Multi` you can create requests that have specialized return values.
```

Comment:
In order to make sure newcomers to the library don't jump straight to this module and get frustrated, I think it makes sense to start this documentation saying something like `This is an advanced version of the Porter module that covers ... For most use-cases, using Porter should be sufficient. If, however, you have a tangible issue with expressing chaining and transformations with your port messages, read along :).`

I feel like your use-case is on the advanced side, and it would be nice to make sure people don't jump in straight away unless it gives them a real benefit.

Reply:
This change has been made. I hope the wording I ended up using meets your expectations :-).